test/verify/check-machines

```diff
@@ -35,33 +35,55 @@ def readFile(name):
 
 @skipImage("Atomic cannot run virtual machines", "fedora-atomic", "rhel-atomic", "continuous-atomic")
 class TestMachines(MachineCase):
-    def testBasic(self):
+    vm1_name="subVmTest1"
+    vm1_id="#vm-{0}".format(vm1_name)
+    vm1_img = "/var/lib/libvirt/images/{0}.img".format(vm1_name)
+
+    def envSetup(self):
         b = self.browser
         m = self.machine
 
-        # We race with running the test after boot but before the libvirt network has started
-        m.execute("virsh net-start default 2> /dev/null || true")
+        # Ensure everything has started correctly
+        m.execute("systemctl start libvirtd")
+        # Wait until we can get a list of domains
+        wait(lambda: m.execute("virsh list"))
+        # Wait for the network 'default' to become active
+        wait(lambda: m.execute(command="virsh net-info default | grep Active"))
 
         self.login_and_go("/machines")
         b.wait_in_text("body", "No VM is running or defined on this host")
 
-        vm1_name="subVmTest1"
-        vm1_id="#vm-{0}".format(vm1_name)
-        vm1_img = "/var/lib/libvirt/images/{0}.img".format(vm1_name)
+        m.execute("qemu-img create -f qcow2 {0} 1G".format(self.vm1_img))
+        m.execute("virt-install --cpu host -r 128 --pxe --force --nographics --noautoconsole --disk path={0},size=1,format=qcow2 -n {1} || true".format(self.vm1_img, self.vm1_name))
+
+    def testBasic(self):
+        self.envSetup()
+
+        b = self.browser
+        m = self.machine
 
-        m.execute("qemu-img create -f qcow2 {0} 256M".format(vm1_img))
-        m.execute("virt-install -r 128 --pxe --force --nographics --noautoconsole -f {0} -n {1}".format(vm1_img, vm1_name))
+        # state == 'running' is expected, but can be 'paused' if the VM creation fails. Anyway, the VM shall be listed at least
+        state = m.execute("virsh domstate {0}".format(self.vm1_name)).rstrip().strip()
```

Comment:
`.rstrip()` is redundant here
I don't think it's that important, but someone wishes to, they can fixup during merge.